mobilecoind/src/conversions.rs

```diff
@@ -298,6 +327,24 @@ mod test {
         let rust = Outlay {
             receiver: public_addr.clone(),
             value: 1234,
+            tx_private_key: None,
+        };
+        let proto = api::Outlay::from(&rust);
+
+        assert_eq!(proto.value, rust.value);
+        assert_eq!(
+            PublicAddress::try_from(proto.get_receiver()).unwrap(),
+            public_addr
+        );
+
+        // Proto -> Rust
```

Comment:
suggest using separate test case, easier to isolate failures, easier to parallelize execution, prevents the test turning into a large workflow/recipe style test
```suggestion
    }

    #[test]
    fn test_outlay_conversion_with_tx_private_key() {
        let mut rng: StdRng = SeedableRng::from_seed([1u8; 32]);
        let public_addr = AccountKey::random(&mut rng).default_subaddress();

```